build.gradle

```diff
@@ -1351,6 +1351,50 @@ project(':tools') {
   }
 }
 
+project(':shell') {
```

Comment:
Is "shell" the right name for this? I think "metadata shell" or "metashell" (like the branch name) is more descriptive of what it does. A Kafka "shell" makes me think of a tool that can do any of the normal command line things to a Kafka cluster (create topics, modify configs, etc). 

Do we ever intend to make this utility more than a read-only metadata explorer?

Maybe we can call it something like "kafka-metadata.sh" or "kafka-metadata-shell.sh"? I'm curious what others think here as well.

Reply:
Good point.  Let's call the command "kafka-metadata-shell.sh"

For the internal names, I think "shell" is fine (and simple) until we have a second shell, if we ever do.  We can always rename it then since it's all internal classes, no API.